watertap/data/techno_economic/agglom_stacking.yaml

```diff
@@ -0,0 +1,8 @@
+default:
+  recovery_vol:
+    value: 0.9999
+    units: dimensionless
+    reference: .nan
```

Comment:
This occurs a few places. Given this is a string field I'd be happier with nearly anything else.

Reply:
That is on the list of things to clean up, but I was thinking of doing that on a case-by-case basis as we did the unit models.